### ISSUE TITLE
Additional zeek module log files

### ISSUE THREAD

**Issue by 111andre111** [elastic/beats#18868]
**Describe the enhancement:**
I checked again the existing log types that exist in filebeat because of a test I made with zeek 3.0.
https://docs.zeek.org/en/current/script-reference/log-files.html
These issues
https://github.com/elastic/beats/issues/12724
https://github.com/elastic/beats/pull/12812
https://github.com/elastic/beats/pull/14150
https://github.com/elastic/beats/pull/14404

I did now produce a list of all logs to identify all missing log types:
* [ ] barnyard2.log
* [ ] broker.log
* [ ] cluster.log
* [ ] config.log
* [ ] known_certs.log
* [ ] known_hosts.log
* [ ] known_modbus.log
* [ ] known_services.log
* [ ] loaded_scripts.log
* [ ] modbus_register_change.log
* [ ] netcontrol_catch_release.log
* [ ] netcontrol_drop.log
* [ ] netcontrol_shunt.log
* [ ] netcontrol.log
* [ ] notice_alarm.log
* [ ] ntp.log
* [ ] openflow.log
* [ ] packet_filter.log
* [ ] print.log
* [ ] prof.log
* [ ] reporter.log
* [ ] software.log
* [ ] stderr.log
* [ ] stdout.log
* [ ] unified2.log
* [ ] weird_stats.log

One special part is extra
* [ ] signatures.log [was already committed](https://github.com/elastic/beats/tree/8e0e31900da39bf4033039c08478b6a97135b910/x-pack/filebeat/module/zeek/signatures) but nevermade it into [release](https://github.com/elastic/beats/tree/v7.7.0/x-pack/filebeat/module/zeek) somehow unfortunately.

And here

**Comment by andrewkroh** [comment]
The signatures.log fileset was never actually merged into Filebeat. I've opened a PR to fix the default config. We'll have to extract the signatures fileset from the closed PR and bring it into master. https://github.com/elastic/beats/pull/18878

**Comment by legoguy1000** [comment]
`signature` and `ntp` have been merged

**Comment by rwaight** [comment]
This issue is relevant and should be worked on.  The current version of Zeek (`4.2.0`) has a list of [65 `.log` files](https://docs.zeek.org/en/current/script-reference/log-files.html).

**Comment by 111andre111** [comment]
Yes @rwaight that would be really neat if we could have some progress here.
